documentation/documentation.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+        <!-- =========================== Favicon ========================= -->
+        <link
+            rel="shortcut icon"
+            href="/assets/images/favicon.png"
+            type="image/x-icon"
+        />
+
+        <!-- ======================== Font Awesome ======================= -->
+        <script
+            src="https://kit.fontawesome.com/04cdd1a366.js"
+            crossorigin="anonymous"
+        ></script>
+
+        <!-- ============================= CSS =========================== -->
+        <link rel="stylesheet" href="/styles/style.css" />
+
+        <title>Kal UI</title>
+    </head>
+    <body>
+        <!-- =========================== Header ========================== -->
+        <header class="header container" id="header">
+            <nav class="nav">
+                <a href="#" class="nav__logo">Kal UI</a>
+
+                <div class="nav__menu" id="nav-menu">
+                    <ul class="nav__list">
+                        <li class="nav__item">
+                            <a href="/index.html" class="nav__link">Home</a>
+                        </li>
+                        <li class="nav__item">
+                            <a
+                                href="documentation/documentation.html"
+                                class="nav__link"
+                                >Documentation</a
+                            >
+                        </li>
+                    </ul>
+                </div>
+                <div class="nav__btns">
+                    <a
+                        href="https://github.com/nerdyspook/Kal-UI-Component-Library"
+                        ><i class="fab fa-github" id="github-btn"></i>
```

Comment:
Just suggestion..
Can add target="_blank" for external links.. so that they open in new tab

Reply:
Thanks for the suggestion, I will do that for all the links which need this.